### PR TITLE
Create centralised utilities and add camelcase handling

### PR DESCRIPTION
**Pull request by Tedowski**

### CODE REVIEW

File: server/src/models/Blog/Collection.model.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    lowercase: true
+  },
+  created_at: {
+    type: Date,
+    default: Date.now
+  },
+  updated_at: {
+    type: Date,
+    default: Date.now
+  }
```
Comment (cichyadam) [inline server/src/models/Blog/Collection.model.js:20]: I think we should be more consistent with naming as I have used camel case naming like createdAt, updatedAt . Either change the BlogPost schema to created_at and updated_at or vice versa

File: server/src/models/Gallery/Image.model.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ref: 'Collection'
+  }],
+  created_at: {
+    type: Date,
+    default: Date.now
+  },
+  updated_at: {
+    type: Date,
+    default: Date.now
+  }
```
Comment (cichyadam) [inline server/src/models/Gallery/Image.model.js:29]: Same as the one above

File: server/src/models/Gallery/Image.model.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  url: {
+    type: String,
+    required: true,
+    index: { unique: true }
+  },
+  collections: [{
+    type: Schema.Types.ObjectId,
+    ref: 'Collection'
+  }],
```
Comment (cichyadam) [inline server/src/models/Gallery/Image.model.js:21]: I'm not use if I understand what is collection used for, we should probably take a call about it.

**Review by cichyadam — CHANGES_REQUESTED** [review]

File: server/src/models/Blog/Collection.model.js (hunk 1 above)
Comment (Tedowski) [reply server/src/models/Blog/Collection.model.js:20]: snake_case will be standard in db layer (mongoose shemas)

**Pull request closed without merging**